packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -195,6 +195,7 @@ class ChartTradeClass extends React.Component {
         const { show_digits_stats, main_barrier, should_refresh, extra_barriers = [] } = this.props;
 
         const barriers = main_barrier ? [main_barrier, ...extra_barriers] : extra_barriers;
+        const active_symbols = ['synthetic_index', 'forex', 'indices', 'stocks', 'commodities'];
```

Comment:
why are we hardcoding this here?

Reply:
It is meant to be reused anywhere else. I think it's better to declare it as local constant here

Reply:
why can't we get it from the API? it's not encouraged to have hardcoded values in FE

Reply:
Sure. I'll make the changes. Thanks!